### ISSUE TITLE
tkn version returns empty string if .kube/config is not preent

### ISSUE THREAD

**Issue by SaffatHasan** [tektoncd/cli#1439]
# Versions and Operating System

- Kubernetes version:

  **Output of `kubectl version`:**

```
/ # kubectl version
Client Version: version.Info{Major:"1", Minor:"19", GitVersion:"v1.19.0", GitCommit:"e19964183377d0ec2052d1f1fa930c4d7575bd50", GitTreeState:"clean", BuildDate:"2020-08-26T14:30:33Z", GoVersion:"go1.15", Compiler:"gc", Platform:"linux/amd64"}
The connection to the server localhost:8080 was refused - did you specify the right host or port?
```

- Tekton Pipeline version:

  **Output of `tkn version` or `kubectl get pods -n tekton-pipelines -l app=tekton-pipelines-controller -o=jsonpath='{.items[0].metadata.labels.version}'`**

```
/ # tkn version
```

(or if I mount my $HOME/.kube)

```

/ # tkn version
Client version: 0.19.0
Pipeline version: unknown, pipeline controller may be installed in another namespace please use tkn version -n {namespace}
```

- Operating System:

 ```
MacOS
 ```

# Expected Behavior

Tekton version is output, even if not connected to kubernetes. Especially when specifying `--component=client`

# Actual Behavior

Tekton output is empty

# Steps to Reproduce the Problem

1. Build the following Docker Image
```
FROM alpine:3.11 as download-binaries

RUN apk add curl

ARG KUBECTL_VERSION=1.19.0

RUN curl -Lk https://storage.googleapis.com/kubernetes-release/release/v${KUBECTL_VERSION}/bin/linux/amd64/kubectl -o kubectl && chmod +x kubectl

ARG TKN_VERSION=0.19.0

RUN ARCH=$(uname -m) && \
	wget --no-check-certificate -O- https://github.com/tektoncd/cli/releases/download/v${TKN_VERSION}/tkn_${TKN_VERSION}_Linux_${ARCH}.tar.gz | \
	tar zxf -

FROM alpine:3.11

COPY --from=download-binaries kubectl	/usr/local/bin/kubectl
COPY --from=download-binaries tkn 	/usr/local/bin/tkn
```
2.  `docker run --rm -it -v"$HOME/.kube:/root/.kube" ${IMAGE}:${VERSION} tkn version`

# Additional Info

**Comment by digglife** [comment]
Yes. That's the current behavior. It's intentional. But as you said, I guess showing the client version and the information about missing configuration file should be more friendly.

Here is the relevant code:

https://github.com/tektoncd/cli/blob/622401ce860593f1a7f8d846664aa7b10acbabae/pkg/cmd/version/version.go#L64-L67

**Comment by digglife** [comment]
Actually it's already fixed in `v0.20.0`. You can download it from [release page](https://github.com/tektoncd/cli/releases/tag/v0.20.0) and try again.

It seems that the homebrew version is still `v0.19.0`, which is not normal.

**Comment by SaffatHasan** [comment]
Updating the version in my Dockerfile resolved the issue.

I'm not sure about the homebrew version as it seems I was already on the `v0.20.0` when I tried installing:

```
➜ brew install tektoncd-cli
Warning: tektoncd-cli 0.20.0 is already installed and up-to-date.
To reinstall 0.20.0, run:
  brew reinstall tektoncd-cli
```
